### PR TITLE
Titania Minor FIxes

### PR DESCRIPTION
**Pull request by VSEphpbb**
Fixes https://tracker.phpbb.com/browse/CUSTDB-756

Also addresses debug notices caused by https://github.com/phpbb/customisation-db/pull/212

### CODE REVIEW

**Comment by DavidIQ** [comment]
Does this address these reports?
https://tracker.phpbb.com/browse/CUSTDB-756
https://tracker.phpbb.com/browse/CUSTDB-757

**Comment by VSEphpbb** [comment]
No it just addresses debug errors because STRIP no longer is defined in phpBB 3.2

**Comment by VSEphpbb** [comment]
OK it does fix issue 756

File: SETUP.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Titania
+
+This is a quick setup guide to walk you through the process of installing and configuring Titania.
+
+## Installation
+
+1. Create a phpBB board.
+   
+   The easiest is option is to use [QuickInstall](https://www.phpbb.com/customise/db/official_tool/phpbb3_quickinstall/) to create a phpBB board. Make sure your config.php is set to Production mode.
```
Comment (rxu) [inline SETUP.md]: Probably, too many ```is``` here? 😃

**Pull request merged**